articles/aks/node-autoprovision.md

```diff
@@ -64,11 +64,23 @@ NAP is based on the Open Source [Karpenter](https://karpenter.sh) project, and t
     ```
 
 ## Limitations
-
-- Windows and Azure Linux node pools aren't supported yet
-- Kubelet configuration through Node pool configuration is not supported
-- NAP can only be enabled on new clusters currently
-- It is not currently possible to stop nodepools or clusters which use the NAP feature
+### Unsupported features:
+- Windows and Azure Linux node pools
+- Kubelet configuration through Node pool configuration
+- IPv6 clusters
+- [Service Principals](/azure/devops/integrate/get-started/authentication/service-principal-managed-identity)
+  - *Note: must be MSI-based, either system managed identity or user managed identity*
+- Disk encryption sets
+- CustomCATrustCertificates
+- [Start Stop mode](./start-stop-cluster.md)
+- [AAD Pod Identity](https://github.com/Azure/aad-pod-identity) (deprecated)
+- [HTTP proxy](./http-proxy.md)
+- [OutboundType](./egress-outboundtype.md) mutation (all OutboundTypes are supported, you just cannot change them once created)
+
+### Other limitations
+- The only network configuration allowed is Cilium + Overlay + Azure
+- Cannot be enabled in a cluster in which **any** AgentPools have autoscaler enabled
```

Comment:
Suggest rewording this sentence to improve grammar to - You can't enable in a cluster where node pools have cluster autoscaler enabled.